src/dashboard/src/components/rights/views.py

```diff
@@ -194,6 +199,9 @@ def rights_edit(request, uuid, id=None, section='ingest'):
             createdRights = form.save()
 
         copyrightFormset = CopyrightFormSet(request.POST, instance=createdRights)
+        if not copyrightFormset.is_valid():
+            return render(request, 'rights/rights_edit.html', locals())
+
```

Comment:
Add this for the other formsets? eg statute_formset, license_formset, other_formset